src/super_gradients/recipes/quantization_params/default_tensorrt_ptq.yaml

```diff
@@ -0,0 +1,34 @@
+quantizer:
+  TRTPTQQuantizer:
```

Comment:
I don't understand the logic of decoupling the exported from the quantizer...
I feel this is puts more risk for errors (like having the TRT quantizer with int8 and onnx exporter, which migt mislead users to try and run that onnx file on onnruntime(.
There is some benefit for debugging though, I guess...